### ISSUE TITLE
External Harvester doesn't seem to work?

### ISSUE THREAD

**Issue by philipwilk** [harvester/harvester#1971]
TL;DR: Nodedriver does not connect to Harvester when External Harvester is selected.

- Can confirm this is not a firewall issue because the other Rancher node I've been trying to use Harvester from has its firewall disabled, unless Harvester is. 
- I have been trying using port 8443 which is the default (that's what the api from Internal Rancher and Internal Harvester shows)

Subsets of this Issue:
1. I'm trying to use this machine driver from within Rancher, it appears as though external harvester doesn't work at all. I have copied the fields directly from the yaml from the internal rancher to the external one, only changing to use the ip address for the harvester node instead of the default hostname, but rancher still can't communicate with rancher. 

2. I'm not sure if this is related, but in rancher the UI resets to internal rancher when editing a node template even though external rancher is selected (but the yaml from api seems to indicate that this is only visual and it does in fact save because the ip I typed in is saved).

3. This is definitely related, absolutely no context is provided in the Rancher interface when using external rancher: the image and network drop-downs are just empty text input boxes instead, so it appears the node driver isn't even connecting to Harvester and getting this information.

**Comment by futuretea** [comment]
Can you provide the versions of docker-machine-driver-harvester ,rancher and harvester?
The following combinations should work:
1. docker-machine-driver-harvester 0.2.0 + rancher 2.6+ harvester 0.2.0
2. docker-machine-driver-harvester 0.1.6 + rancher 2.5+ harvester 0.2.0

**Comment by philipwilk** [comment]
I've tried machine-driver .1.6 with rancher 2.5.9 and harvester .2.0 and it hasn't worked.
I've just a build of Rancher 2.6.0 which has it included and harvester .2.0 and once I've copied the config for the node template from the internal Rancher, it appears to 'work' as far as making nodes (but it's not acutally doing it properly because the three nodes I've made have not started (over an hour now) ). 

However, image and network are still empty inputs so it is impossible to provision nodes without first going to harvester or internal rancher and finding out the id for the image (which just doesn't work because it's like 'image-2pvwq' or similar which you just can't remember/know without checking every time) 
![Screenshot_20210812_224648](https://user-images.githubusercontent.com/50517631/129274540-8328f57d-3e66-48e7-8901-19fdab91033b.png)
